assets/js/phoenix_live_view.js

```diff
@@ -870,7 +871,6 @@ export let DOM = {
 
   disableForm(form, prefix){
     let disableWith = `${prefix}${PHX_DISABLE_WITH}`
```

Comment:
We also need to move the disabled with to inside  "data ref". I wonder if the data ref handling could be made generic such that you simply attach an anonymous function to a data ref and the given function is always applied after a morphdom patch. This would make it easy to handle disabled with and css classes under the same conceptual model. It would also help when apply data ref to inputs too!